### PR TITLE
Catch exceptions also when loading the available extensions (bsc#941491)

### PR DESCRIPTION
**Pull request by lslezak**
- 3.1.141

Do not crash when an exception is raised.

(Note: An error popup is displayed internally inside the `ConnectHelpers.catch_registration_errors` it's enough to just check the return value).

### CODE REVIEW

**Comment by ancorgs** [comment]
Nothing added to `inst_scc_test.rb`? :wink:

File: test/inst_scc_test.rb [hunk 1]
```diff
@@ -28,6 +28,16 @@
       expect(Yast::WFM.call("inst_scc")).to eq(:abort)
     end
 
+    it "displays an error when loading the available extensions fails" do
```
Comment (ancorgs) [inline test/inst_scc_test.rb]: Well, strictly speaking you are not checking that the error is displayed. You are simply checking that the whole thing does not explode... which is actually nice to know (and proves the bug to be fixed) but not what the predicate says. :-)

**Comment by ancorgs** [comment]
LGTM

**Comment by lslezak** [comment]
Thanks, let's merge #200 first...

**Pull request merged**